pymove/utils/math.py

```diff
@@ -1,6 +1,27 @@
 import math
 
 
+def is_number(value):
+    """
```

Comment:
Codacy found an issue: [First line should end with a period, question mark, or exclamation point (not 'n') (D415)](https://app.codacy.com/gh/InsightLab/PyMove/pullRequest?prid=6012588)